rsql-common/src/main/java/io/github/perplexhub/rsql/RSQLCustomPredicateInput.java

```diff
@@ -15,16 +16,18 @@ public class RSQLCustomPredicateInput {
     CriteriaBuilder criteriaBuilder;
     Path<?> path;
     Attribute<?, ?> attribute;
+    String attributeName;
```

Comment:
Given that we already have field named `attribute` don't you find this `attributeName` name misleading?

Reply:
In case of a Jsonb you don't have the endpoint in the jsonb but the column name.